spec/features/repair_request_spec.rb

```diff
@@ -43,7 +43,7 @@ def stub_post_repair_request
     )
   end
 
-  def stub_keyfax_get_startup_url
+  def stub_keyfax_session_create
```

Comment:
Not that it is super important, but the old naming was better. This is a stub to the network call, not the `KeyfaxSession.create` method.

Reply:
changed them back 👍